crates/core/src/lib.rs

```diff
@@ -58,7 +58,7 @@ fn dotenv(name: impl AsRef<Path>) -> Result<Option<PathBuf>, dotenv::Error> {
 #[derive(Debug, Clone, Copy, clap::Parser)]
 pub struct ServerOpts {
     /// The port to listen on
-    #[clap(short, long, default_value_t = 3000, env)]
+    #[clap(short, long, default_value_t = 3000, env = "PORT")]
```

Comment:
```suggestion
    #[clap(short, long, default_value_t = 3000, env)]
```
The `= "PORT"` should be implicit, was this added back for a reason?